tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-django22-celery{44}-drf39,py38-django{22,30}-celery{44}-drf{39,310,311,312}
+envlist = py{3.8,3.9}-django{22,30,31}-celery{44}-drf{39,310,311,312},quality,docs
```

Comment:
At first I'd assumed the `38` -> `3.8` switch was needed to simplify the GitHub Action, but it doesn't look that way.  Was there a specific need for that change?  The Travis test config still says `py38`, this isn't causing any trouble there?

Reply:
you're right, that was my initial understanding, turned out to be a bit more intuitive than I first thought.